lib/publiccloud/provider.pm

```diff
@@ -256,7 +269,17 @@ sub terraform_apply {
         $image, $args{count}, $instance_type, $self->region);
 
     assert_script_run($cmd);
-    assert_script_run('terraform apply myplan', TERRAFORM_TIMEOUT);
+    my $ret = script_run('terraform apply myplan', TERRAFORM_TIMEOUT);
+    unless (defined $ret) {
+        type_string(qq(\c\\));        # Send QUIT signal
+        sleep 1;                      # give some time to Quit
+        assert_script_run('true');    # make sure we have a prompt
```

Comment:
ah this might work actually . what if we will just run 
```
my $tries = 3;
while(script_run('true') && $tries){ 
$tries = tries -1;
sleep 1` 
}
```

Reply:
I will give it a try, thx

Reply:
Maybe we can just remove the `sleep 1;` I actually never tried. But https://github.com/os-autoinst/os-autoinst/blob/master/distribution.pm#L126 already has a more suitable sleep in there.
WDYT @asmorodskyi  ?

Reply:
would be fine too , and if we will start getting trouble here we will implement something as above later